package.json

```diff
@@ -1,49 +1,49 @@
 {
-    "name": "redux-fetcher",
-    "version": "1.0.0-alpha1",
-    "description": "Simple isomorophic fetch for Redux",
-    "main": "index.js",
-    "scripts": {
-        "build": "babel src --out-dir lib --source-maps",
-        "build:watch": "npm run build -- --watch",
-        "clean": "rimraf lib docs coverage",
-        "test": "npm run lint && npm run test:coverage",
-        "test:unit": "mocha --compilers js:babel/register --recursive test/",
-        "test:unit:watch": "npm run test:unit -- --watch",
-        "test:coverage": "babel-node ./node_modules/.bin/isparta cover _mocha -- --recursive test/",
-        "lint": "eslint .",
-        "check": "ncu",
-        "docs": "esdoc -c esdoc.json",
-        "prepublish": "npm test && npm run clean && npm run build && npm run docs"
-    },
-    "repository": {
-        "type": "git",
-        "url": "git+https://github.com/vgno/redux-fetcher.git"
-    },
-    "author": "Andreas Søvik <arsovik@gmail.com>",
-    "license": "MIT",
-    "bugs": {
-        "url": "https://github.com/vgno/redux-fetcher/issues"
-    },
-    "homepage": "https://github.com/vgno/redux-fetcher#readme",
-    "devDependencies": {
-        "babel-eslint": "~4.1.1",
-        "esdoc": "~0.4.3",
-        "esdoc-es7-plugin": "0.0.3",
-        "esdoc-importpath-plugin": "0.0.1",
-        "chai": "^3.4.1",
-        "mocha": "^2.3.4",
-        "eslint": "~1.9.0",
-        "eslint-config-vgno": "~5.0.0",
-        "eslint-plugin-react": "~3.9.0",
-        "sinon": "~1.17.2",
-        "sinon-chai": "~2.8.0",
-        "rimraf": "~2.4.4",
-        "isparta": "~3.5.3"
-    },
-    "dependencies": {
-        "babel": "~5.8.34",
-        "babel-runtime": "^5.1.11",
-        "redux-api-middleware": "github:vgno/redux-api-middleware#v1.0.0-beta4"
-    }
+  "name": "redux-fetcher",
+  "version": "1.0.0-alpha1",
+  "description": "Simple isomorophic fetch for Redux",
+  "main": "./lib/index.js",
+  "scripts": {
+    "build": "babel src --out-dir lib --source-maps",
+    "build:watch": "npm run build -- --watch",
+    "clean": "rimraf lib docs coverage",
+    "test": "npm run lint && npm run test:coverage",
+    "test:unit": "mocha --compilers js:babel/register --recursive test/",
+    "test:unit:watch": "npm run test:unit -- --watch",
+    "test:coverage": "babel-node ./node_modules/.bin/isparta cover _mocha -- --recursive test/",
+    "lint": "eslint .",
+    "check": "ncu",
+    "docs": "esdoc -c esdoc.json",
+    "prepublish": "npm test && npm run clean && npm run build && npm run docs"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/vgno/redux-fetcher.git"
```

Comment:
Maybe use the same type of URL as we have for the Roc modules, that being a normal url:
```json
"repository": {
    "type": "git",
    "url": "https://github.com/vgno/roc-web-react.git"
  }
```